client/gutenberg/extensions/presets/jetpack/utils/jetpack-block-type.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { every, has, get, pickBy } from 'lodash';
+
+/**
+ * Where Jetpack data can be found in WP Admin
+ */
+const JETPACK_DATA_PATH = [ 'Jetpack_Initial_State' ];
+
+export default class JetpackBlockType {
+	constructor( name, config ) {
+		this.name = name;
+		this.config = config;
+		this.requiredModules = null;
```

Comment:
Probably a lot of this boilerplate (and the accompanying methods) will go away with a wp.data approach - we'll have all these just as `props`.